Description: Add Fedora packaging
 Add Fedora/EL packaging information into rpm/ directory.
 .
 packaging-test (0.2-1) unstable; urgency=medium
 .
   * Initial release
Author: Jakub Ru탑i훾ka <voice@texnoforge.dev>

---
Last-Update: 2020-07-11

--- /dev/null
+++ packaging-test-0.2/rpm/demo-libuv.service
@@ -0,0 +1,6 @@
+[Unit]
+Description=demo_libuv oneshot service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/demo_libuv
--- /dev/null
+++ packaging-test-0.2/rpm/packaging-test.spec
@@ -0,0 +1,47 @@
+Name:           packaging-test
+Version:        0.2
+Release:        1%{?dist}
+Summary:        Packaging Test package
+
+License:        GPLv3
+URL:            https://gitlab.nic.cz/knot/packaging-test
+Source0:        https://gitlab.labs.nic.cz/knot/packaging-test/-/archive/v%{version}/%{name}-v%{version}.tar.gz
+Source1:        demo-libuv.service
+
+BuildRequires:  gcc
+BuildRequires:  libuv-devel
+
+Requires:       libuv
+
+%description
+This is a testing package to prove my mad packaging skills.
+
+It contains demo_libuv binary and demo-libuv oneshot systemd service.
+
+
+%prep
+%setup -qn %{name}-v%{version}
+
+%build
+gcc -g -luv demo_libuv.c -o demo_libuv
+
+%install
+install -p -D -m 0755 demo_libuv %{buildroot}%{_bindir}/demo_libuv
+install -p -D -m 644 %{SOURCE1} %{buildroot}%{_unitdir}/demo-libuv.service
+
+%post
+%systemd_post demo-libuv.service
+
+%preun
+%systemd_preun demo-libuv.service
+
+
+%files
+%license LICENSE
+%{_bindir}/demo_libuv
+%{_unitdir}/demo-libuv.service
+
+
+%changelog
+* Sat Jul 11 2020 Jakub Ru탑i훾ka <voice@texnoforge.dev> - 0.2-1
+- Initial Release
